01UCS_LAL-UCLA/js/custom.js

```diff
@@ -647,14 +647,6 @@
     var lc = document.createElement('script'); lc.type = 'text/javascript';
     lc.src = 'https://v2.libanswers.com/load_chat.php?hash=libanswers_secret_86530858';
     var s = document.getElementsByTagName('script')[0]; s.parentNode.append(lc);
-    setTimeout(()=>{
```

Comment:
@ztucker4 Is there any negative to removing this `setTimeout()` function?  This seems to... wait 2 seconds to confirm the widget exists, and then position it if it does exist.  Everything seems to work without it, I'm just unsure of the consequences.  You mentioned redundancy; does everything needed come from the script loaded via `https://v2.libanswers.com/load_chat.php`?

Reply:
@akohler No negatives that I can identify. All that happens in that function is setting styling, which ends up not being visible after the styling from the script from `load_chat`. I took a look at a couple of other UCs' Primo pages using the LibChat button, and any additional styling there is coming from the custom CSS file. I'm not sure why this function was added in the first place.